blockstore/apps/bundles/models.py

```diff
@@ -74,7 +74,7 @@ class Collection(models.Model):
     """
     id = models.BigAutoField(primary_key=True)
     uuid = models.UUIDField(default=uuid.uuid4, unique=True, editable=False)
-    title = models.CharField(max_length=MAX_CHAR_FIELD_LENGTH, db_index=True)
+    title = models.CharField(max_length=MAX_CHAR_FIELD_LENGTH, db_index=True, db_collation='utf8mb4_general_ci')
```

Comment:
@bradenmacdonald Turns out changing the collation also updates the table to use the associated character set:
https://dev.mysql.com/doc/refman/8.0/en/charset-table.html

So this worked!